server/mailet/integration-testing/src/test/java/org/apache/james/mailets/utils/IMAPMessageReader.java

```diff
@@ -75,7 +75,7 @@ public String readFirstMessageInInbox(String user, String password) throws IOExc
     public boolean userGetNotifiedForNewMessages(int numOfMessage) throws IOException {
         imapClient.noop();
 
-        return imapClient.getReplyString().contains(numOfMessage + " RECENT");
```

Comment:
Anyway, we did not added enough details on this....

The reply string was : 

```
 * 1 RECENT
 * 1 EXISTS
TAG OK noop completed.
```

And : 

```
 * 1 EXPUNGED
TAG OK noop completed.
```

I performed these checks more in depth... As well as rebase on master.